helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -119,6 +121,14 @@ internal fun findRepositories(directory: File): Map<String, VcsInfo> {
     return ortResult.repository.nestedRepositories
 }
 
+/**
+ * Build the file for the split curations.
+ */
+internal fun getSplitCurationFile(parent: File, packageId: Identifier, fileExtension: String) =
```

Comment:
Other bits of the "helper-cli" use `Identifier.toPath()`. See "generate package config" command.
Would it make sense to re-use it here as well? (A small modification is needed to exclude the version from the path)

Reply:
This is intentionally not used.
See the discussion here: https://github.com/oss-review-toolkit/ort/pull/4141#discussion_r648406981

Reply:
For package configuration, I believe '_' is used instead of 'unknown'. Would that make sense here as well (for consistency) ?

Reply:
Reference to the package configuration:
https://github.com/oss-review-toolkit/ort/blob/f8d90dee54e1e6cdca5d0a6bed99e92bdec5e650/helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt#L109

For consistency this makes sense, yes.
But I don't really like the resulting directory structure if using `_` as parent folder if the namespace is unknown.

For multiple package manager that will mean that everything is moved to the "useless" folder.
e.g. every `Crate`, `GoMod` and most `NPM` dependencies will be moved into a `_` folder.